src/PrestaShopBundle/Form/Admin/Sell/Order/Delivery/SlipOptionsType.php

```diff
@@ -49,15 +49,31 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 TranslatableType::class,
                 [
                     'type' => TextType::class,
+                    'label' => $this->trans('Delivery prefix', 'Admin.Orderscustomers.Feature'),
+                    'help' => $this->trans('Prefix used for delivery slips.', 'Admin.Orderscustomers.Help'),
                 ]
             )
             ->add(
                 'number',
-                FormType\NumberType::class
+                NumberType::class,
+                [
+                    'label' => $this->trans('Delivery number', 'Admin.Orderscustomers.Feature'),
+                    'help' => $this->trans(
+                        'The next delivery slip will begin with this number and then increase with each additional slip.',
+                        'Admin.Orderscustomers.Help'
+                    ),
+                ]
             )
             ->add(
                 'enable_product_image',
-                SwitchType::class
+                SwitchType::class,
+                [
+                    'label' => $this->trans('Enable product image', 'Admin.Orderscustomers.Feature'),
```

Comment:
Do you think it would be possible to replace 'Enable product image' with 'Product image' and change 'Yes' and 'No' to 'Enabled' and 'Disabled' (See screenshot)

I think it is better for user experience. You don't have to read the verb in the label to understand the meaning of the 'Yes' and the 'No'.  We are trying to harmonize the wording for the status toggles in the software. So it would be great if we could start implementing it as we go along with PRs. If it is not possible, I will create a dedicated issue. Thanks!

![image](https://user-images.githubusercontent.com/70583503/111297851-67160700-864e-11eb-97c2-22abf116b165.png)